src/store/favorite/favorite.slice.js

```diff
@@ -0,0 +1,48 @@
+import { createSlice } from '@reduxjs/toolkit'
+import {
+   deleteFavoriteItems,
+   getFavoriteItems,
+   postFavoriteItem,
+} from './favorite.thunk'
+
+const initialState = {
+   favoriteItems: [],
+   isLoading: false,
+}
+
```

Comment:
Cannot read properties of null (reading 'length')
TypeError: Cannot read properties of null (reading 'length')